play/src/front/Phaser/Game/GameScene.ts

```diff
@@ -1980,6 +1980,7 @@ export class GameScene extends DirtyScene {
             if (availabilityStatus === AvailabilityStatus.SILENT) {
                 this.CurrentPlayer.toggleTalk(false, true);
             }
+            this.chatConnection.setPresence(availabilityStatus);
```

Comment:
We could listen to the `availabilityStatusStore` in the MatrixChatConnection. Even better: we could inject the `availabilityStatusStore` in the constructor if the `MatrixChatConnection`